### PR TITLE
Change not to expire accessToken

### PR DESCRIPTION
**Pull request by murata100**

### CODE REVIEW

File: azure.go [hunk 1]
```diff
@@ -100,6 +103,7 @@ func (ac *AzureClient) getAccessToken() {
 		log.Fatalf("Error unmarshalling response body: %v", err)
 	}
 	ac.accessToken = data["access_token"].(string)
+	ac.accessTokenCreatedAt = time.Now().UTC()
```
Comment (brian-brazil) [inline azure.go]: It might be a bit cleaner to store when it expires rather than when it was created

Comment (murata100) [reply azure.go]: I feel I should use `data["expires_on"]`.

https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service#service-to-service-access-token-response

Comment (brian-brazil) [reply azure.go]: I meant more moving around the existing logic

Comment (murata100) [reply azure.go]: changed

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**